src/bioregistry/data/bioregistry.json

```
@@ -60469,6 +60474,7 @@
       ]
     },
     "license": "CC0-1.0",
+    "logo": "https://interpro-documentation.readthedocs.io/en/latest/_static/interpro_logo_text.svg",
```

```suggestion
    "logo": "https://content.embl.org/sites/default/files/03-2025/interpro_newlogo-2-scaled-1.jpg",
```

This one will work better since the svg is white text only.